src/main/java/core/basesyntax/model/User.java

```diff
@@ -25,10 +24,9 @@
 public class User {
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
-    @Column(name = "id", nullable = false)
     private Long id;
     private String username;
 
-    @OneToMany(cascade = CascadeType.PERSIST,fetch = FetchType.EAGER)
+    @ManyToMany(cascade = CascadeType.PERSIST, fetch = FetchType.EAGER)
```

Comment:
Why do we use ManyToMany here?

Reply:
Because I am thinking that one message is owned by many users, and their commits under the message are owned by user.